app/services/subscribe_service.rb

```diff
@@ -5,15 +5,21 @@ def call(account)
     account.secret = SecureRandom.hex
 
     subscription = account.subscription(api_subscription_url(account.id))
-    response = subscription.subscribe
+    response     = subscription.subscribe
 
-    unless response.successful?
+    if response.code > 299 && response.code < 500 && response.code != 429
```

Comment:
Maybe wrap this up in a well named method and remove the comments? - `response_failed?(response.code)` or something...